src/crypto.c

```diff
@@ -156,14 +156,16 @@ int bip32_CKDpub(const serialized_extended_pubkey_t *parent,
 }
 
 void crypto_ripemd160(const uint8_t *in, uint16_t inlen, uint8_t out[static 20]) {
-    cx_ripemd160_hash(in, inlen, out);
+    int res = cx_ripemd160_hash(in, inlen, out);
+    LEDGER_ASSERT(res == CX_OK, "Unexpected error in ripemd160 computation");
 }
 
 void crypto_hash160(const uint8_t *in, uint16_t inlen, uint8_t out[static 20]) {
     PRINT_STACK_POINTER();
 
     uint8_t buffer[32];
-    cx_hash_sha256(in, inlen, buffer, 32);
+    int res = cx_hash_sha256(in, inlen, buffer, 32);
+    LEDGER_ASSERT(res == CX_SHA256_SIZE, "Unexpected error in sha256 computation");
```

Comment:
It could be nice to add the res value in the log message :woman_shrugging:

Reply:
Done in f4d38edfcf4a2da5e55ff9813fe7d61abc6677c8.

Might be worth adding that in the boilerplate examples.